code/modules/reagents/chemistry/reagents/toxin_reagents.dm

```diff
@@ -163,16 +163,19 @@
 	toxpwr = 0.5
 	taste_description = "death"
 
-/datum/reagent/toxin/zombiepowder/on_mob_metabolize(mob/living/L)
-	..()
-	L.fakedeath(type)
-
 /datum/reagent/toxin/zombiepowder/on_mob_end_metabolize(mob/living/L)
 	L.cure_fakedeath(type)
 	..()
 
 /datum/reagent/toxin/zombiepowder/on_mob_life(mob/living/carbon/M)
 	M.adjustOxyLoss(0.5*REM, 0)
+	switch(current_cycle)
+		if(1 to 5)
+			M.confused += 1
+			M.drowsyness += 1
+		if(5 to INFINITY)
+			M.fakedeath(type)
+			. = 1
```

Comment:
?

Reply:
maybe go from 6 to INFINITY instead of 5 to INFINITY?

Reply:
should it mute for the first 10 seconds, or would that be too strong?

Reply:
this was cargo cult